environment.yml

```diff
@@ -17,6 +17,7 @@ dependencies:
   - r-base
   - r-rcpp
   - libgfortran-ng # needed by r packages
+  - hdf5
   - cdo  
```

Comment:
I need cdo=1.9.3 here for this to work

Reply:
You are right, I noted this above. Is 1.9.5 working in other branches? I switched to 1.9.3 for an error related to missing libhdf5.so.103.